Do not arbitrarily use linebreaks in comments (#23)
invenia/BlueStyle: README.md

Always make a priority of keeping the comments up-to-date with code changes!

Comments should be complete sentences.

> Long sentences should be wrapped at 92 characters and not earlier.

Reviewer: What if the 92'd character is midword?
What if its mid-quote?
what if there is a comma at character 80?

> Moving to a newline for a new sentence is okay.

Reviewer: ```suggestion
Except if a new sentence is being started.
```

When referencing Julia in documentation note that "Julia" refers to the programming language while "julia" (typically in backticks, e.g. `julia`) refers to the executable.

> Only use inline comments if they fit within the line length limit. If your comment comment cannot be fitted inline then place the comment above the content to which it refers.

Reviewer 2: I prefer the colon here. I think we're rather inconsistent in this document though.
